plugins/jira-dashboard/README.md

```diff
@@ -61,6 +61,23 @@ const defaultEntityPage = (
 );
 ```
 
+3. OPTIONAL - List of user issues on the homepage can be enabled by this code in your `app/src/components/home/Homepage.tsx` :
+
+![user-issues-list.png](media%2Fuser-issues-list.png)
+
+```tsx
+ <Grid item xs={12} md={6}>
+    <JiraUserIssuesViewCard
+      bottomLinkProps={{
+       link: 'https://our-jira-server/issues',
+       title: 'Open in Jira',
+     }}
+   />
+ </Grid>
+```
+Note that the list of user issues is limited by permissions defined for the token used by backend.
+The username is being extracted from the user's email (everything before @). 
```

Comment:
Maybe refer to the token, and the JIRA_EMAIL_SUFFIX with a link to: https://github.com/AxisCommunications/backstage-plugins/edit/main/plugins/jira-dashboard-backend/README.md#configuration-details

Then the reader will know what you are talking about :)

Reply:
Doc updated